lms/envs/common.py

```diff
@@ -1654,13 +1654,15 @@ def _make_locale_paths(settings):
         'source_filenames': [
             'certificates/css/main-ltr.css',
             'css/vendor/font-awesome.css',
+            'css/edraak_certificate.css',
```

Comment:
Please add the corresponding empty `edraak_certificate.scss` file so `paver update_assets` don't break when there's no theme.

Reply:
Done